webapp/src/widgets/footer/Footer.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import { Container } from 'react-bootstrap'
+
+const Footer = () => (
+	<footer className='py-30'>
+		<Container fluid>
+			<span className='f-14'>Copyright 2020 Mapright</span>
```

Comment:
use JS to get current year or we will have to update it after each new year

Reply:
updated.